website/docs/developer/research/research-reconciliation.mdx

```diff
@@ -776,18 +776,27 @@ The ORCA inventory catalog build script is available for download <a href={useBa
 The ORCA internal reconciliation tables build script is available for download <a href={useBaseUrl('files/build_orca_reconcile.sql')} target="_blank" download>here</a>.
 
 ## Notes from ORCA schema refinement research
-As part of the research, some refinements were made to the ORCA inventory catalog schema. Some attribute modifications were made only to the `granules` table:
+As part of the research, some refinements were made to the ORCA inventory catalog
+schema. This includes refinements to the `granules` table and removal of the
+`provider_collection_xref` table and the relationship between provider and
+collection.
+
 - `execution_id` has been added for helping with logging and tracing.
+- `cumulus_create_time` has been added for time based filtering with cumulus. The cumulus `createdAt` time will be a consistent time source between catalogs
+- `provider_id` has been added to create a relationship between granule and provider.
 - `archive_location` has been removed since `orca_archive_location` already exists in the `files` table.
 
 #### Refined Granule Metadata Object
 
-| Attribute Name   | Data Type | Description                                                            | Notable Items   |
-| ---------------- | --------- | ---------------------------------------------------------------------- | --------------- |
-| cumulus_granule_id | `string`  | granuleId that is provided by Cumulus.                                 | Must be unique per collectionId |
-| execution_id       | `string`  | Step function execution ID from AWS.                                  | Unique ID automatically generated by AWS |
-| ingest_time      | `timestamp` | Date and time in UTC that the data was originally ingested into ORCA | None |
-| last_update      | `timestamp` | Date and time in UTC that information was updated.                   | Potential time filter for Cumulus reconciliation. |
+| Attribute Name      | Data Type | Description                                                            | Notable Items   |
```

Comment:
Guess we were missing some stuff here. Would be nice to bring the column markers in line with each-other.